FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -348,7 +348,11 @@ - (void)retryHTTPConnection {
   __weak RCNConfigRealtime *weakSelf = self;
   dispatch_async(_realtimeLockQueue, ^{
     __strong RCNConfigRealtime *strongSelf = weakSelf;
-    if (strongSelf->_remainingRetryCount > 0) {
```

Comment:
wdyt about adding `if (strongSelf->_isInBackground) { return; }` at the beginning instead?